### PR TITLE
Release 2022.01.2

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Release Notes for 2022.01.2
- **Is this PR related to a Monolith release?** Yes. 2022.01.2

### Resolves:

* [OKTA-460422](https://oktainc.atlassian.net/browse/OKTA-460422)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 1]
```diff
@@ -6,6 +6,22 @@ title: Okta Identity Engine API Products release notes 2022
...
 
+### Weekly release 2022.01.2
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.01.2](#bugs-fixed-in-2022-01-2)         | January 26, 2022           |
+
+#### Bugs fixed in 2022.01.2
+
+* When the [Create a new Binding](/docs/reference/api/roles/#create-a-new-binding) or the [Add more Members to a Binding](/docs/reference/api/roles/#add-more-members-to-a-binding) endpoints were called, and included all users or all groups in the request, the request didn't fail as expected. (OKTA-459994)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:17]: Should we call these "endpoints"? There is just one endpoint, with different request types. Maybe use "Resource Set operations"?

**Review by vanngo-okta — APPROVED** [review]
Just a style question. Otherwise, everything looks good! 🚀

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 2]
```diff
@@ -6,6 +6,32 @@ title: Okta Identity Engine API Products release notes 2022
...
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Device information in the OAuth 2.0 interact flow](#device-information-in-the-oauth-2-0-interact-flow) | January 26, 2022 |
+| [Fewer digits revealed for shorter phone numbers](#fewer-digits-revealed-for-shorter-phone-numbers) | January 26, 2022 |
+| [Bugs fixed in 2022.01.2](#bugs-fixed-in-2022-01-2)         | January 26, 2022           |
+
+#### Device information in the OAuth 2.0 interact flow
+
+Confidential clients can now specify device information using the `X-Device-Token` header during the OAuth 2.0 `/interact` flow. <!--OKTA-455553-->
+
+#### Fewer digits revealed for shorter phone numbers
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:21]: Can we add some context? At first glance, I didn't know what this refers to. Maybe: 
```suggestion
#### Fewer phone number digits revealed in an Authentication response
```

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 3]
```diff
@@ -6,6 +6,32 @@ title: Okta Identity Engine API Products release notes 2022
...
+
+The masking algorithm now reveals fewer digits for shorter phone numbers. For example, if the phone number has eight digits, the first five digits are masked and the final three digits are visible. <!--OKTA-455393-->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:23]:
```suggestion
The masking algorithm now reveals fewer digits for shorter profile phone numbers in an [authentication response](/docs/reference/api/authn/#response-example-for-primary-authentication-with-public-application-success). For example, if the phone number has eight digits, the first five digits are masked and the final three digits are visible. <!--OKTA-455393-->
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 4]
```diff
@@ -3,6 +3,23 @@ title: Okta API Products release notes 2022
...
 ## January
 
+### Weekly release 2022.01.2
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Fewer digits revealed for shorter phone numbers](#fewer-digits-revealed-for-shorter-phone-numbers) | January 26, 2022 |
+| [Bugs fixed in 2022.01.2](#bugs-fixed-in-2022-01-2)         | January 26, 2022           |
+
+#### Fewer digits revealed for shorter phone numbers
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:13]:
```suggestion
#### Fewer phone number digits revealed in an Authentication response
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 5]
```diff
@@ -3,6 +3,23 @@ title: Okta API Products release notes 2022
...
+
+The masking algorithm now reveals fewer digits for shorter phone numbers. For example, if the phone number has eight digits, the first five digits are masked and the final three digits are visible. <!--OKTA-455393-->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:15]:
```suggestion
The masking algorithm now reveals fewer digits for shorter profile phone numbers in an [authentication response](/docs/reference/api/authn/#response-example-for-primary-authentication-with-public-application-success). For example, if the phone number has eight digits, the first five digits are masked and the final three digits are visible. <!--OKTA-455393-->
```

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 6]
```diff
@@ -6,6 +6,32 @@ title: Okta Identity Engine API Products release notes 2022
...
 
+### Weekly release 2022.01.2
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Device information in the OAuth 2.0 interact flow](#device-information-in-the-oauth-2-0-interact-flow) | January 26, 2022 |
+| [Fewer digits revealed for shorter phone numbers](#fewer-digits-revealed-for-shorter-phone-numbers) | January 26, 2022 |
+| [Bugs fixed in 2022.01.2](#bugs-fixed-in-2022-01-2)         | January 26, 2022           |
+
+#### Device information in the OAuth 2.0 interact flow
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:17]: We don't document the /interact API for OIE? Should this be in the release notes?

**Review by vanngo-okta — COMMENTED** [review]
I don't think the /interact API is documented yet on our site?
Also, I added a link to /authn response for the phone number masking feature, not sure if it's correct. Please confirm with dev.

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md (hunk 2 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:21]: Shaye doesn't want to include that change as it affects more than just the authentication.

**Pull request merged**